run-build-functions.sh

```diff
@@ -14,6 +14,11 @@ export GIMME_CGO_ENABLED=true
 export NVM_DIR="$HOME/.nvm"
 export RVM_DIR="$HOME/.rvm"
 
+# Pipenv configuration
+export PIPENV_RUNTIME=3.6
```

Comment:
I think I would prefer this to be `2.7` if possible, since that is our default python version at the moment.